docs/architecture/posixfs.md

```diff
@@ -0,0 +1,173 @@
+---
+title: "Posix Filesystem"
+date: 2024-05-27T14:31:00+01:00
+weight: 30
+geekdocRepo: https://github.com/owncloud/ocis
+geekdocEditPath: edit/master/docs/architecture
+geekdocFilePath: posixfs.md
+---
+
+{{< toc >}}
+
+Posix FS is the working name for the collaborative storage driver for Infinite Scale.
+
+The scope of this document is to give an high level overview to the technical aspects of the Posix FS and guide the setup.
+
+## Introduction
+
+Posix FS is a backend component that manages files on the server utilizing a "real" file tree that represents the data with folders and files in the file system as users are used to it. That is the big difference compared to Decomposed FS which is the default storage driver in Infinite Scale.
```

Comment:
Looking into the dev docs and searching for `decomposed`, I only find [Decomposed FS on NFS](https://owncloud.dev/ocis/storage-backends/dcfsnfs/). We should add a new documentation about Decomposed FS, what it is about and link that document here for reference so one can compare. This is especially true as Decomposed FS uses POSIX (eg NFS) for metadata stored via messagepack and this new FS driver uses XATTRs as part of the files stored.